snippets/functions/boilerplate/generalFunction.js

```diff
@@ -0,0 +1,47 @@
+exports = async function(arg){
+    // This default function will get a value and find a document in MongoDB
+    // To see plenty more examples of what you can do with functions see:
+    // https://www.mongodb.com/docs/atlas/app-services/functions/
+   
+   
+    // Find the name of the MongoDB service you want to use (see "Linked Data Sources" tab)
+    var serviceName = "mongodb-atlas";
```

Comment:
using hardcoded  service, db and Coll names here to match the other snippets in this repo